app/controllers/va_facilities_controller.rb

```diff
@@ -1,16 +1,35 @@
 class VaFacilitiesController < ApplicationController
   include PracticeUtils
   before_action :set_va_facility, only: [:show, :created_practices, :update_practices_adopted_at_facility]
-
+  helper_method :crh_created_practices_count, :crh_adopted_practices_count
```

Comment:
can we make a `va_facilities_helper.rb` file and move this and its associated methods out of the controller

Reply:
Yep - moved methods to va_facilities_helper.rb file.